optimum/intel/neural_compressor/modeling_base.py

```diff
@@ -63,6 +63,14 @@
 """
 
 
+if is_intel_extension_for_transformers_available():
+    from intel_extension_for_transformers.transformers.modeling import AutoModelForCausalLM as ITREX_WOQ_MODEL
+
+    class ITREXAutoModelForCausalLM(ITREX_WOQ_MODEL):
+        auto_model_class = AutoModelForCausalLM
+        export_feature = "text-generation"
+
+
```

Comment:
ITREX support should be part of `INCModel` as it was done before https://github.com/huggingface/optimum-intel/pull/455/commits/0cc7c0065d1bb9c901c916d93e2971f892233ef1

Reply:
Yes, I had integrated WOQ model into INCModel.